libs/log-formats.js

```diff
@@ -17,9 +19,9 @@ const type2color = {
 function prodFormat (type, message, label, namespace) {
   // properly display error messages
   if (message.stack) message = message.stack
-  if (message.toString) message = message.toString()
 
-  return JSON.stringify({ time: new Date(), type, message, label, namespace })
+  // cut the string to avoid a fail in the stack
+  return JSON.stringify({ time: new Date(), type, message, label, namespace }).substr(0, LOG_LENGTH_LIMIT)
```

Comment:
I think you should cut the message by message.substr(0, LOG_LENGTH_LIMIT - type.length - label.length - namespace.length) because here we will end up with an invalid JSON.

Reply:
il faut pas hésiter aussi à retirer un nombre de caractère en plus pour les `" { } :`

Reply:
Pour moi on est dans un cas limite qui ne doit pas arriver (une ligne de log du plus de 64K) et si dans ce cas, on se retrouve avec un JSON invalide, ça apparaitra quand même dans les logs en tant que log que la stack n'a pas réussi à parser et je trouve ça pas plus mal.

En tout cas, c'était volontaire mais discutable.

Reply:
ta solution me va aussi et comme tu le dis ça doit être exceptionnel ;)

Reply:
OK